Tests/Server.py

```diff
@@ -217,6 +217,22 @@ def test_get_transaction_log_entries_from_today(self):
             entry_date = entry_timestamp.date()
             today_date = datetime.date.today()
             self.assertTrue(entry_date == today_date)
+            
+    @unittest.skip("Doesn't work in TM1 11")
```

Comment:
I think the `skip` decorator here and the one above can be removed. 
I remember I added for one of the early releases of TM1 11, when the transaction log filtering was broken in TM1.

But now since it works I think we shouldn't skip the test.

Reply:
Enabled below three tests:
`test_get_last_transaction_log_entries`
`test_get_transaction_log_entries_from_today`
`test_get_transaction_log_entries_until_yesterday`